lib/workos/sso.rb

```diff
@@ -124,6 +124,25 @@ def client
         @client
       end
 
+      sig { params(path: String).returns(Net::HTTP::Post) }
+      def post_request(path:)
```

Comment:
Going to extract all of this into a common client setup file for use with the audit log module